integration-test/src/main/scala/com/cloudera/livy/test/framework/BaseIntegrationTestSuite.scala

```diff
@@ -97,17 +97,18 @@ abstract class BaseIntegrationTestSuite extends FunSuite with Matchers {
     }
   }
 
-  test("initialize test cluster") {
+  override def beforeAll() {
```

Comment:
I did this because when I run a specific test e.g. `mvn -Dsuites="com.cloudera.livy.test.InteractiveIT user"`
Test fails because test("initialize test cluster") isn't executed.

Since this is less trivial than it looks, I will do this in a separate CR.